cluster/manifests/skipper/deployment.yaml

```diff
@@ -301,6 +301,9 @@ spec:
 {{ if .Cluster.ConfigItems.skipper_ingress_inline_routes }}
           - "-inline-routes={{ .Cluster.ConfigItems.skipper_ingress_inline_routes }}"
 {{ end }}
+{{ if .Cluster.ConfigItems.skipper_ingress_passive_health_check_options }}
```

Comment:
I don't think that's correct

Reply:
I don't know which problem you had in your mind, but I see the following, imagine:
+ we merge this PR (only canary)
+ I set `skipper_ingress_passive_health_check_options` config item to something like `period=1s,min-requests=10,max-drop-probability=0.9`
+ CLM sets this option both to canary and main fleet
+ main fleet version is older that version where PHC was introduced -> main fleet cannot pars cmdline option -> main fleet doesn't roll out

Reply:
If you set it to `""` it's likely also `true`, not sure but I think we do not use the implicit empty string test. I think we use `index .foo > 0` or something like that. Please check it.

Reply:
> we do not use the implicit empty string test

We literally do this three lines above this line:
```
{{ if .Cluster.ConfigItems.skipper_ingress_inline_routes }}
          - "-inline-routes={{ .Cluster.ConfigItems.skipper_ingress_inline_routes }}"
{{ end }}
```

As far as I see, inline routes work for us for years without problem, so I expect the option provided in this PR to work the same way.

Reply:
thanks